### PR TITLE
Cleaned the previous PR

### PR DESCRIPTION
**Pull request by nprabhune**

### CODE REVIEW

**Comment by nprabhune** [comment]
@CharlesAHunt Sorry for uploading all the files. I have cleaned it and created this new PR.
Please have a look!
Thanks,
Neha

File: challenges/Backend Challenge/app.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@app.route('/')
+def index():
+	return "Hello World!"
+
+@app.route('/data', methods = ['GET'])
+def sortnumbers():
+	if len(arr) == 0:
+		return "Number list is empty"
+	arr.sort()
```
Comment (CharlesAHunt) [inline challenges/Backend Challenge/app.py]: You decided to sort the list in the GET request, which works.  I have a question.  Assume our client will POST a list of numbers once, and then GET that same list of numbers millions of times every day.  How would this change your implementation to minimize computation?  You don't need to write any more code, just respond with a short description of how you would re-design this solution to handle the scenario I described.

File: challenges/Backend Challenge/app.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	res = ""
+	for ele in arr:
+		res += str(ele) + " "
+	return res
```
Comment (CharlesAHunt) [inline challenges/Backend Challenge/app.py]: The description of the challenge asked for JSON formatting of the response. Is this response String JSON formatted?  If not could you complete the PR with code to JSON format the response?

File: challenges/Backend Challenge/app.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+@app.route('/data', methods = ['POST'])
+def validatenumbers():
+	response = requests.request('POST', 'http://localhost:5000/data/list.txt')
+    e = response.json()
+    arr = e['numberList']
+    if(len(arr)<500):
```
Comment (CharlesAHunt) [inline challenges/Backend Challenge/app.py]: It looks like you check for the list being less than 500 elements. The challenge asks for the list to also not exceed 500 numbers.  Do you also check if the array is greater than 500 numbers?

File: challenges/Backend Challenge/app.py (hunk 1 above)
Comment (nprabhune) [reply challenges/Backend Challenge/app.py]: @CharlesAHunt : If the user is just going to POST the number list once, I could sort it then itself, and for every GET call, simply return the sorted list as many as times required in the necessary format!

File: challenges/Backend Challenge/app.py (hunk 2 above)
Comment (nprabhune) [reply challenges/Backend Challenge/app.py]: Please find the changes in app1.py. Thanks.

File: challenges/Backend Challenge/app.py (hunk 3 above)
Comment (nprabhune) [reply challenges/Backend Challenge/app.py]: So sorry I missed that! I have fixed it. Please have a look at app1.py. Thank you.

File: challenges/Backend Challenge/app1.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        return json.dumps(res)
+
+@app.route('/data', methods = ['POST'])
+def validatenumbers():
+        if request.heads['Content-Type']!='application/json':
+                return "Invalid format"
+        response = requests.request('POST', 'http://localhost:5000/data/list.txt')
+        e = response.json()
+        arr = e['numberList']
+        if(len(arr)!=500):
```
Comment (CharlesAHunt) [inline challenges/Backend Challenge/app1.py]: Looks good!

**Pull request closed without merging**